gnucash/report/reports/standard/new-aging.scm

```diff
@@ -415,12 +403,8 @@ exist but have no suitable transactions."))
      (gnc:make-multichoice-option
       gnc:pagename-display optname-addr-source "a" (N_ "Address source.") 'billing
       (list
-       (vector 'billing
-               (N_ "Billing")
-               (N_ "Address fields from billing address."))
-       (vector 'shipping
-               (N_ "Shipping")
-               (N_ "Address fields from shipping address.")))))
+       (vector 'billing (N_ "Address fields from billing address"))
+       (vector 'shipping (N_ "Address fields from shipping address")))))
```

Comment:
Option name itself is "Address Source".  The values can be reduce to "Billing Address" and "Shipping Address" IMO.